### PR TITLE
fix: replace "github.com/pkg/errors" with stdlib "errors" package

### PR DESCRIPTION
**Pull request by justenstall**
**What this PR does / why we need it**:

Replaces the archived "github.com/pkg/errors" package with the Go standard library equivalents.

**Special notes for your reviewer**:

**If applicable**:
- [ ] this PR contains user facing changes (the `docs needed` label should be applied if so)
- [ ] this PR contains unit tests
- [x] this PR has been tested for backwards compatibility

### CODE REVIEW

File: cmd/helm/status.go [hunk 1]
```diff
@@ -140,11 +138,9 @@ func (s statusPrinter) WriteTable(out io.Writer) error {
 		_, _ = fmt.Fprintf(out, "VERSION: %s\n", s.release.Chart.Metadata.Version)
 		_, _ = fmt.Fprintf(out, "APP_VERSION: %s\n", s.release.Chart.Metadata.AppVersion)
 	}
-	if s.showDescription {
-		_, _ = fmt.Fprintf(out, "DESCRIPTION: %s\n", s.release.Info.Description)
-	}
+	_, _ = fmt.Fprintf(out, "DESCRIPTION: %s\n", s.release.Info.Description)
```
Comment (mattfarina) [inline cmd/helm/status.go:141]: This change is breaking some tests and they don't expect the description to be present.

**Review by mattfarina — COMMENTED** [review]
I have not given this much of a review. I did find one change that's causing numerous tests to fail. The expected output needs to be updated so the tests pass.

File: cmd/helm/status.go (hunk 1 above)
Comment (justenstall) [reply cmd/helm/status.go:141]: I accidentally started this branch from #13444, I'll cherry-pick the commits to a new branch and re-open the PR

Comment (justenstall) [reply cmd/helm/status.go:141]: Reopened as #13460

**Pull request closed without merging**